assets/src/modules/BaseLayersMap.js

```diff
@@ -274,6 +276,11 @@ export default class BaseLayersMap extends olMap {
                     name: node.name
                 });
 
+                // OL layers zIndex is the reverse of layer's order given by cfg
+                if (!node.itemState.groupAsLayer){
+                    layer.setZIndex(layersCount - 1 - node.layerOrder);
+                }
```

Comment:
```suggestion
                layer.setZIndex(layersCount - 1 - node.layerOrder);
```
layerOrder is available for group as layer https://github.com/3liz/lizmap-web-client/blob/master/assets/src/modules/state/Layer.js#L1283

Reply:
@rldhont I have this error w/ your suggestion in layer_legends project:

```
TypeError: this.itemState is undefined
    get layerOrder Layer.js:1285
    get layerOrder MapLayer.js:537
    createNode BaseLayersMap.js:281
    createNode BaseLayersMap.js:195
    BaseLayersMap BaseLayersMap.js:293
    uicreated Lizmap.js:70
    triggerEvent OpenLayers.js:255
    init map.js:5453
map.js:5458```